### PR TITLE
Ci taming

### PR DESCRIPTION
**Pull request by jdpye**
adding some CI configurability scripts to help us manage dev builds better.

### CODE REVIEW

**Comment by mhpob** [comment]
TL;DR @jdpye You can probably close this one. I added `testthat::skip_on_ci` in [596064e](https://github.com/ocean-tracking-network/glatos/pull/204/commits/596064eefdfa0c6bf83f7d2beff4b6725c165a86) that will cover 90% of use cases.

More details: The RUniverse GH Action is flagged as CI via `testthat::skip_on_ci` (it has a "CI" env var), so `skip_on_ci` will skip RUniverse's GH Actions as well as those running in this repo. I built the RUniverse skipper since some of my tests use GH secrets and the secrets are not transferred to RUniverse's GH Actions. I wanted it to run on CI within my repo, which was secrets-aware, but skip specifically when RUniverse was running the test, which is not secrets-aware. Unless `glatos` starts using GH Secrets for its CI, there's no reason to include this.

**Comment by jdpye** [comment]
Thanks Mike, I'll fold this one up and look for something else to pick off next :D
